debug/mangle.json

```diff
@@ -8,7 +8,12 @@
       "properties": {
         "regex": "^_",
         "reserved": [
-          "__REACT_DEVTOOLS_GLOBAL_HOOK__"
+          "__REACT_DEVTOOLS_GLOBAL_HOOK__",
+          "_renderers",
+          "emit",
```

Comment:
Only properties with a leading underscore get minified, so these might not be necessary.

Reply:
Good point! Fixed :heavy_check_mark: